### PR TITLE
setup and mobile first

### PR DESCRIPTION
**Pull request by fatima-najafi**
For the first milestone in the process of creating  portfolio website i implement these steps: 
-  The 2 sections  build for this project header and headline.
-  Set up correctly Linter  for this repository.
-  Descriptive README file
- Pull Request is made from 'feature branch' into the main branch.
-  Used HTML/CSS

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        </div>
+       
+      </nav>
+    </header>
+    <!--header section ends-->
+
+    <!--headline section starts-->
+    <section class="headline-main">
+      <section class="headline">
+        <h2 class="headline-heading">I'm Fatima.<br />Glad to see you!</h2>
```
Comment (topeogunleye) [inline index.html]: - [ ] Hi, great job on your web page. I noticed that you used an h2 tag for the main title of your web page (e.g., “I’m Marianne …”). I suggest that you change it to an h1 tag instead. The reason is that an h1 tag is the most important heading on your web page and it tells Google (and other search engines) what your web page is about. In this case, you want Google to know that your web page is about you. Using an h1 tag will also improve the accessibility and readability of your web page for users who use screen readers or have low vision. You can check this link for more info: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements. Please make this change and resubmit your project. Thank you for your cooperation. 😊

File: index.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        <p class="headline-paragraph">
+          I'm a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don't hestiate to contact
+          me.
+        </p>
+        <p class="headline-sm-suport">LET'S CONNECT</p>
```
Comment (topeogunleye) [inline index.html]: - [ ] Hi, nice work on your project. I noticed that you used a p tag for a subtitle on your web page. This is a subtitle for the icons at the bottom of it. I suggest that you use a header tag hx instead, where x is a number from 2 to 6 depending on the level of importance of your subtitle. The reason is that header tags are semantic elements that indicate the structure and hierarchy of your web page. They also help Google (and other search engines) understand the main topics and subtopics of your web page. Using header tags will also improve the accessibility and readability of your web page for users who use screen readers or have low vision. You can check this link for more info: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements. Please make this change and resubmit your project. Thank you for your cooperation. 😊

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @fatima-najafi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Try-Again

> ## (Highlights) Good Points: 💪 
> * Great implementation of Header section. ✔️ 
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:
> * PR has a good title and summary :heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ]  Next time, and for future commits, I kindly suggest that you please continue make use of imperative mood in the subject line for your commit messages as you have done in the rest of your commit messages except the one highlighted in the picture below.
:x: Bad Commit Messages :x:
image

### ✔️ Good Commit Messages: ✔️
✔️  Add fix for dark mode toggle state
✔️  Update README file.
✔️  Add styles to h1.

- [ ] Please refer to this article: please read it and apply it in your work from now on: [Professional repo - basic rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md);

- [ ] For future pull requests, please notice that your branch name `Header-Section` does not follow the convention of being named after a feature or action. For example, you could name your branch `added-Header-Section`. This makes it easier to track the changes and the purpose of each branch.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by jerryowusu — APPROVED** [review]
Hi @fatima-najafi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

200w


## Highlights
:heavy_check_mark: Neat and well organized code
:heavy_check_mark: Great effort in making the requested changes


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**